index.html

```diff
@@ -0,0 +1,152 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" crossorigin="anonymous" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="wrapper">
+    <header class="container header">
+        <div class="flex-wrapper">
+            <h2 class="logo-name">
+                <a href="#" class="logo">Meri-MG</a>
+            </h2>
+            <nav>
+                <a href="#" title="bar-menu"><i class="fas fa-bars"></i></a>
+            </nav>
+        </div>
+    </header>
+    <section class="container headline">
+        <h1>Meri Gogichashvili</h1>
+        <h2>Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work a
+                nd experience! If you like what you see and have a project you need coded, don’t 
+                hesitate to contact me.
+            </p>
+    <hr>
+    <div class="info-wrapper">
+        <ul>
+            <li class="list-header">Language</li>
+            <li class="list">Javascript</li>
+            <li class="list">Ruby</li>
+            <li class="list">Html</li>
+            <li class="list">Css</li>
+        </ul>
+        <ul>
+            <li class="list-header">Frameworks</li>
+            <li class="list">Bootstrap</li>
+            <li class="list">Ruby on Rails</li>
+            <li class="list">RSPeC</li>
+            <li class="list">CapyBara</li>
+            <li class="list">Selenium</li>
+        </ul>
+        <ul>
+            <li class="list-header">Skills</li>
+            <li class="list">Codekit</li>
+            <li class="list">Github</li>
+            <li class="list">Codepen</li>
+            <li class="list">Gitlab</li>
+            <li class="list">Terminal</li>
+        </ul>
+</div>
+    <hr>
+        <div class="flex-icons">
+            <ul>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-github"></i></a>
+                </li>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-linkedin-in"></i></a>
+                </li>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-angellist"></i></a>
+                </li>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-twitter"></i></a>
+                </li>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-medium-m"></i></a>
+                </li>
+            </ul>
+        </div>
+</section> 
+<div class="works-wrapper">
+    <section class="container works">
+        <div class="work-images">
+            <img src="images/Group55.png" alt="down-arrow">
+            <img src="images/Work.png" alt="work">
+        </div>
```

Comment:
If the section has a navigation link (e.g., "Work") it uses an `a` tag.

- [ ] Please, add your Work image inside `a` tag 😉 Remeber to add `href="#"` as the attribute.